evaluation/evaluation_rel.py

```diff
@@ -21,40 +21,33 @@ def calculate_rmse_error(gt_pose, calc_pose):
 
     return translation_errors, translation_error_rmse, rotation_error_degrees, rotation_errors_individual
 
-def main(config_path):
-    config = load_config(config_path)
-    ground_truth = config['ground_truth']
-    calibration = config['calibration']
-    
-    main_lidar_pose = ground_truth['main_lidar']
-    
-    translation_errors_all = []
-    rotation_errors_all = []
-    rotation_errors_individual_all = []
-    
-    for lidar, gt_pose in ground_truth.items():
-        if lidar != 'main_lidar':
-            gt_pose_matrix = np.array(gt_pose)
-            calc_pose = calibration[lidar]
-            translation_errors, translation_error_rmse, rotation_error_degrees, rotation_errors_individual = calculate_rmse_error(gt_pose_matrix, calc_pose)
-            print(f"Errors for {lidar}:")
-            print(f"  Translation Errors [x, y, z]: {translation_errors}")
-            print(f"  Translation Error (RMSE) [m]: {translation_error_rmse}")
-            print(f"  Rotation Error (Degrees): {rotation_error_degrees}")
-            print(f"  Rotation Errors Individual [r, p, y]: {rotation_errors_individual}")
-            print()
-            translation_errors_all.append(translation_errors)
-            rotation_errors_all.append(rotation_error_degrees)
-            rotation_errors_individual_all.append(rotation_errors_individual)
-    
-    avg_translation_errors = np.mean(translation_errors_all, axis=0)
-    avg_rotation_errors = np.mean(rotation_errors_all)
-    avg_rotation_errors_individual = np.mean(rotation_errors_individual_all, axis=0)
+def evaluate(calibration_results: list, sensors_config_path: str):
+    config = load_config(sensors_config_path)
```

Comment:
Add comments here to clearly define that this is the current transform that we are reading to compare against

Reply:
Also change the variable name